onboard/nav/stateMachine.cpp

```diff
@@ -392,10 +386,16 @@ double StateMachine::getOptimalAvoidanceAngle() const
     return mPhoebe->roverStatus().obstacle().bearing;
 } // optimalAvoidanceAngle()
 
+bool StateMachine::waypointReachable( double distance, double bearing ) 
+{
+    return ( ( mPhoebe->roverStatus().obstacle().distance < distance - 1 ) ||
+     ( ( bearing - mPhoebe->roverStatus().odometry().bearing_deg ) < 0 
+                        && mPhoebe->roverStatus().obstacle().bearing > 0 ) ||
+     ( ( bearing - mPhoebe->roverStatus().odometry().bearing_deg ) > 0 
+                        && mPhoebe->roverStatus().obstacle().bearing < 0 ) );
+}
 
 // thresholds based on state? waypoint vs ball
-// set distance to go around obstacles?
-// set threshold for when to skip a point?
 
 // TODOS:
 // [drive to ball] obstacle and ball
```

Comment:
Didn't you fix this `TODO`?

Reply:
yes. The distance to go around obstacles is the distance CV is sending us and we're not missing any waypoints anymore